containers/Dockerfile.ppc64le

```diff
@@ -0,0 +1,29 @@
+# Dockerfile for cerberus for ppc64le arch
+
+FROM ppc64le/centos:8
```

Comment:
@juliemathew This is great. We might want to add document around using an image built using this dockerfile when running Cerberus on Power: https://github.com/cloud-bulldozer/cerberus/blob/master/docs/installation.md#containerized-version. Thoughts?

Reply:
yes, I am working on this @chaitanyaenr

Reply:
I have added required documentation for supporting Cerberus tool on Power arch. 
Please review @chaitanyaenr  .Thank you.